src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.Square;
+import core.basesyntax.supplier.FigureSupplier;
+import java.util.Random;
+
+public class Main {
+    private static final FigureSupplier FIGURE_SUPPLIER = new FigureSupplier();
+
+    public static void main(String[] args) {
+
+        Figure[] figures = new Figure[new Random().nextInt(6)];
+
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = FIGURE_SUPPLIER.getRandomFigure();
+        }
+        figures[3] = new Circle("WHITE", 6);
+        figures[4] = new Square("RED", 4);
+
+        for (Figure figure: figures) {
+            figure.draw();
+        }
+    }
```

Comment:
```suggestion
    public static void main(String[] args) {
         FigureSupplier figureSupplier = new FigureSupplier();

        Figure[] figures = new Figure[6];

        for (int i = 0; i < figures.length; i++) {
            if (i < figures.length / 2) {
                figures[i] = figureSupplier.getRandomFigure();
            } else {
                figures[i] = figureSupplier.getDefaultFigure();
            }
        }

        for (Figure figure: figures) {
            figure.draw();
        }
    }
```
services in main don't have to be class-level variables

your task was to create **6 figures**
first 3 random, the other 3 default